test/webgl-utils/parameters.spec.js

```diff
@@ -0,0 +1,168 @@
+import {createTestContext} from '../setup';
+import {GL} from 'luma.gl';
+import {glSetParameters, glGetParameter, glCopyParameters, GL_PARAMETER_DEFAULTS}
+  from '../../src/webgl-utils/parameters';
+import trackContext from '../../src/webgl-utils/track-context';
```

Comment:
params does not depend on trackContext. It is a bit circular to import trackContext here.